packages/browser-vm/src/proxyInterceptor/shared.ts

```diff
@@ -114,12 +114,12 @@ const buildInProps = makeMap([
   Symbol.hasInstance,
 ]);
 
-function transferProps(o: Function, n: Function) {
-  for (const key of Reflect.ownKeys(o)) {
+function transferProps(source: Function, target: Function) {
```

Comment:
The original parameter naming was too confusing, which led to parameter passing errors. Here it is changed to source and target for easier understanding.